pool/pgpoolstorage/pgpoolstorage.go

```diff
@@ -622,3 +622,19 @@ func (p *PostgresPoolStorage) UpdateTxWIPStatus(ctx context.Context, hash common
 	}
 	return nil
 }
+
+// IsAddressBlocked check if an address is added to the blocked table
+func (p *PostgresPoolStorage) IsAddressBlocked(ctx context.Context, address common.Address) (bool, error) {
```

Comment:
The idea is to load the full list during startup. It won't be a long list and can be kept in memory.

There must be a go func reloading this is memory each 5 / 10 minutes.